run.py

```diff
@@ -89,7 +101,15 @@ def run(command, env={}):
             # So we need to make sure we call layout.get with the correct argument names:
             myKwarg = {"datatype" : modality} if modality in ['anat','func','fmap'] else {"suffix" : modality}
             # get filenames matching:
-            myImages = layout.get(subject=subject_label,
+            if session_to_analyze is not "":
+                print("Session: %s"% session_to_analyze)
+                myImages = layout.get(subject=subject_label,
+                                  session=session_to_analyze,
+                                  **myKwarg,
+                                  extensions=["nii.gz", "nii"],
+                                  return_type='file')
+            else:
+                myImages = layout.get(subject=subject_label,
```

Comment:
Thanks for adding this functionality!
I would take advantage of the myKwargs to add the "session" argument there, if present:
```
            if session_to_analyze is not "":
                print("Session: %s"% session_to_analyze)
                myKwarg['session']=session_to_analyze
            myImages = layout.get(subject=subject_label,
```

Reply:
Thank you for fast feedback, and suggestion for improvement. I have added your suggestion into the last commit of the pull request.